packages/shared/components/organisms/ProposalCard.svelte

```diff
@@ -53,7 +53,8 @@
     <div class="flex justify-between items-center">
         <ProposalStatusInfo status={proposalState?.status} milestones={proposal.milestones} />
         {#if hasVoted}
-            <TooltipIcon icon={Icon.Voted} size="small" position={Position.Left} iconClasses="text-gray-500">
+            {@const icon = proposal.status === ProposalStatus.Commencing ? Icon.History : Icon.Voted}
+            <TooltipIcon {icon} size="small" position={Position.Left} iconClasses="text-gray-500">
```

Comment:
This is not necessary as far as I can see from the designs.

Reply:
Ok, just added this because it was in the checklist of the issue